product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/health_status.mdx

```diff
@@ -0,0 +1,71 @@
+- A schematic view of all the nodes organized in a cluster distributed across regions displaying health and role(s) of each node.
+- A replication status view displaying the status of each replication slot and the associated replication lags.
+
+## Viewing Health Status dashboard
+
+To view the **Health Status** dashboard from the BigAnimal portal:
+
+1. In the left navigation of BigAnimal Portal, go to **Clusters**.
+2. Select any ready high-availability or PGD cluster.
+3. On the cluster detail page, select the **Health Status** tab.
+
+The **Health Status** tab displays the dashboard with health status categorized in sections:
+- [Viewing Health Status dashboard](#viewing-health-status-dashboard)
+  - [Global cluster health](#global-cluster-health)
+  - [Regional nodes health and roles](#regional-nodes-health-and-roles)
+  - [Replication status](#replication-status)
+
+### Global cluster health
+
+The global cluster health section displays the cluster-wide view including the following metrics:
+
+- **Raft Status** (PGD only)&mdash; It indicates whether the Raft consensus algorithm is running correctly in the cluster. It verifies that one node is elected as the global leader and the Raft roles such as RAFT_LEADER and RAFT_FOLLOWER are defined correctly across all the nodes.
+- **Replication Slot Status** (PGD only)&mdash; It indicates whether all the replication slots are in streaming state or not.
+- **Clock Skew**&mdash; It indicates whether the node's clock is in sync and doesn't exceed a threshold of 60 seconds.
+- **Proxy Status** (PGD only)&mdash; It provides the number of PGD proxies up and running as compared to the available proxies.
+- **Node Status**&mdash; It provides the number of nodes that are up and running.
+- **Transaction rate**&mdash; It provides the total number of transactions including committed and rolled back transactions per second in the cluster.
+
+
+### Regional nodes health and roles
+
+The regional nodes health and roles section displays fine-grained health status at regional and node level. It is structured as an accordion, with each element representing a group of nodes deployed in the same region. Each item displays basic information like:
+- **Proxy Status** (only PGD)&mdash; It indicates the number of active proxies compared to the available proxies in the specified regions.
+- **Node Status** &mdash; The number of nodes up and running as compared to the available nodes in the specified region in a text chart. It provides the status of all nodes in the specified region using boolean indicator (green (OK)/red (KO)) in a text chart.
+
+On expanding each item, it provides a list nodes with information like:
+- Total number of active connections as compared to the maximum number of configured connections for each node.
+- Memory usage percentage on a progress bar.
+- Storage usage percentage on a progress bar.
```

Comment:
We recently added a "Node Ko" tag at node level that appears whenever the node is down. Can you please add this information?

Reply:
@aviola, can you share the screenshot of this?